k8s/dust-kube-private/services/socks-proxy-service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: socks-proxy-service
+spec:
+  type: ClusterIP
+  ports:
+  - port: 1080
+    targetPort: 1080
+  selector:
+    app: socks-proxy
+---
+apiVersion: net.gke.io/v1
+kind: ServiceExport
```

Comment:
This exports to the other cluster via MCS (kept in the same file as suggested by google's doc)

Reply:
IMO it would be more consistent with our setup to have a separate folder for "service exports"

but loosely held